youtube_dl/extractor/olympicchannel.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import ExtractorError, try_get, unified_strdate
+
+
+class OlympicChannelIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?olympicchannel\.com/../playback/(?P<id>.+)'
+    _TEST = {
+        'url': 'https://www.olympicchannel.com/en/playback/snowboard-mens-halfpipe',
+        'info_dict': {
+            'id': 'epis_63b711050c9749ab8ca3a6fa07f4cff6',
+            'ext': 'mp4',
+            'title': 'Men\'s Halfpipe Finals - Snowboard | PyeongChang 2018 Replays',
+            'description': 'The men\'s halfpipe finals were held at the Phoenix Snow Park on 14 February 2018.',
+            'duration': 6801,
+            'release_date': '20160601',
+            'thumbnail': r're:^https?://.*\.jpg$',
+        }
+    }
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+        data = self._parse_json(
+            self._html_search_regex(r'data-widget-data=["\']([^"\']+)["\']', webpage, 'data'), display_id)
+        item = data['items'][0]
+
+        video_id = data['uid']
+        title = data.get('title') or self._og_search_title(webpage)
+        description = data.get('synopsis') or self._og_search_description(webpage)
+        duration = item.get('duration_in_seconds')
```

Comment:
`int_or_none`.

Reply:
After redesign, there is no more duration.